installing/getting-started-rhacs-cloud-ocp.adoc

```diff
@@ -119,4 +119,30 @@ If no data appears in the ACS Console:
 * Ensure that at least one secured cluster is connected to your {product-title-managed-short} instance. For more information, see the "Installing secured cluster resources on each cluster" section.
 * Examine your Sensor pod logs to ensure that the connection to your {product-title-managed-short} instance is successful.
 * In the {ocp-short} cluster, navigate to *Platform Configuration* -> *Clusters* to verify that the components are healthy and view additional operational information.
-* Examine the values in the `SecuredCluster` API in the Operator on your local cluster to ensure that the *Central API Endpoint* has been entered correctly. This value should be the same value as shown in the *ACS instance* details in the {cloud-console}.
\ No newline at end of file
+* Examine the values in the `SecuredCluster` API in the Operator on your local cluster to ensure that the *Central API Endpoint* has been entered correctly. This value should be the same value as shown in the *ACS instance* details in the {cloud-console}.
+
+[id="default-access-acs-console"]
+== Default access to ACS Console
+
+By default, the only authentication mechanism available to users is authentication via Red Hat SSO.
+Red Hat SSO auth provider cannot be deleted or changed.
+However, you can change minimum access role and add additional rules.
+
+[NOTE]
+====
+In case you want more control over access to your ACS console, it is recommended not to rely on default auth provider.
+You can set up your own IdP provider right after provisioning using one of the guides under `Operating -> Managing user access`
+====
+
+Default auth provider has required attribute `rh_org_id` set to the organisation ID assigned to your account.
```

Comment:
```suggestion
The default authentication provider has the required attribute `rh_org_id` set to the organization ID assigned to your account.
```

Reply:
Let's not call it "default" anywhere in this doc because no such concept exists. Instead, I suggest "preconfigured RHSSO" or "built-in" or something along the lines.

Reply:
"your organization", "the organization" terms are vague. What we speak about here is organization in terms of Red Hat SSO. I suggest we introduce this concept above. Let's make it clear here.

Also "your account" is vague. It's the account of the user who created the ACSCS instance.

Reply:
Regarding the "organization" term: I wonder if it should be explained somewhere earlier in the process. When users access https://console.redhat.com/beta/application-services/acs/instances, they see ACS instances of other users within the same organization. Explaining what organization means on the stage when ACS is already created might be too late.

Saying that, with the help of the IT team I found the next definition of "organization account": https://source.redhat.com/groups/public/ciams/docs/topic_external_sso_enablements#attributes-needed, see `Organization account ID`. I reused it partially within this paragraph - it does not feel right to create a separate subsection just for two sentences.